firmware/hw_layer/mc33816_data.c

```diff
@@ -51,7 +51,7 @@ const unsigned short MC33816_data_RAM[128] =
 const unsigned short MC33816_main_config[29] =
 {
     0x0003, 0x1FFE, 0x0000, 0x1200, 0x0000, 0x0000, 0x0001, 0x0000, 0x001F, 0x0000,
-    0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000,
+    0x0000, 0x0000, 0x0000, 0x0000, 0x0001, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000,
```

Comment:
what does setting this bit do?

Reply:
Reset_behavior register (0x1CE)
bit 0: Driver enable reset behavior (driver_enable_rb): if set to ‘1’ Driver_status register is reset on read.

Reply:
Ah, neat.